bundle/Resources/views/themes/ngadmin/content/location_view.html.twig

```diff
@@ -26,7 +26,13 @@
                     </svg>
                 </a>
                 <a href="{{ url }}">{{ content_type.name }}</a>
+
+                {% set array = [] %}
```

Comment:
One final nitpick, this variable should be named something like `languages` or `language_names`

Reply:
Done